src/lib/components/Menu.svelte

```diff
@@ -61,25 +63,35 @@
 
     // Shift the menu on large screen e.g. if a banner is displayed
     @include media.min-width(large) {
-      padding: calc(
-          var(--menu-logo-height) + var(--padding-3x) +
-            var(--header-offset, 0px) - var(--menu-selection-outer-radius)
-        )
-        var(--padding-2x) 0;
-      // remove extra space because of menu selection touches the edge
-      padding-right: 0;
+      padding-top: calc(
+        var(--menu-logo-height) + var(--padding-3x) + var(--header-offset, 0px) - var(
+            --menu-selection-outer-radius
+          )
+      );
     }
   }
 
   .inner {
     display: flex;
     flex-direction: column;
-    gap: var(--padding-0_5x);
 
     // More space for menu selection touches the edge;
     // otherwise the first selected menu entry would be cut off in mobile view.
     padding-top: var(--menu-selection-outer-radius);
 
+    .slot-content {
+      display: flex;
+      flex-direction: column;
+
+      gap: var(--padding-0_5x);
+
+      padding-left: var(--padding);
```

Comment:
As you are cleaning maybe it would be ideal to store this paddings at top as a variable also like 91 and use them in line 120 and line 113 .

Then we will be building on single variable for padding left.

Not necessary but would be nice